I	O
uploaded	O
it	O
and	O
got	O
an	O
internal	O
error	O
Premature	O
end	O
of	O
script	O
After	O
much	O
playing	O
around	O
,	O
I	O
found	O
that	O
if	O
I	O
had	O
import	O
cgiimport	O
cgitb	O
;	O
import	O
cgitb.enable	B
(	I
)	I
It	O
would	O
give	O
me	O
a	O
much	O
more	O
useful	O
answer	O
and	O
say	O
that	O
it	O
was	O
not	O
installed	O
,	O
you	O
can	O
see	O
it	O
yourself	O
-	O
>	O
http	O
:	O
//woarl.com/db.py	O
Oddly	O
enough	O
,	O
this	O
would	O
produce	O
an	O
error	O
import	O
MySQLdbimport	O
cgiimport	O
cgitb	O
;	O
cgitb.enable	B
(	I
)	I
I	O
looked	O
at	O
some	O
of	O
the	O
other	O
files	O
I	O
had	O
up	O
there	O
and	O
it	O
seems	O
that	O
library	O
was	O
one	O
of	O
the	O
ones	O
I	O
had	O
already	O
tried	O
.	O
